hack/prepare-supervisor-on-kind.sh

```diff
@@ -534,6 +592,12 @@ if [[ "$use_ad_upstream" == "yes" ]]; then
   echo
 fi
 
+if [[ "$use_github_upstream" == "yes" ]]; then
+  echo "    GitHub Username: your github username"
```

Comment:
May want to change what we say here.

Reply:
I'm not sure we need this at all. There's no login flow that would ever require them to provide the CLI with their credentials.